pkg/multiTenancyPlugins/naming/nameScoping.go

```diff
@@ -150,15 +152,15 @@ func CheckContainerReferences(cluster cluster.Cluster, tenantId string, containe
 	var err error
```

Comment:
we can't remove  'var err error' since later it is used:
if containerReferenceToIdMap, err = getIDsFromContainerReferences(cluster, tenantId, containerRefs); err != nil {
		return err
	}